### PR TITLE
Add Documentation for Connectors Overview and HTTP and MQTT

### PR DESCRIPTION
**Pull request by nicholastmosher**
This PR contains the documentation update for the Connectors Overview, and the HTTP and MQTT connectors. This should be the minimal set of documentation necessary to publish the first iteration of connectors.

Contents include:

- Connector Overview
  - The stages of connectors (protocol, extract, filter, shape)
  - Local Connectors
  - Managed Connectors
  - Smart Connectors

- HTTP
  - Overview
  - Configuration
  - Launching as Local Connector
  - Launching as Managed Connector
  - Event data and samples

- MQTT
  - Overview
  - Configuration
  - Launching as Local Connector
  - Launching as Managed Connector
  - Event data and samples

Feedback is welcome!

### CODE REVIEW

File: content/connectors/sources/http.md [hunk 1]
```diff
@@ -1,3 +1,99 @@
...
+continuously, and building streaming applications that react to new or updated info.
+Note that this connector is _not_ intended for streaming HTTP endpoints, it instead
+periodically sends HTTP requests and collects the response body as an event.
+
+## Configuration Options
+
+The HTTP connector supports the following configuration options:
+
+- `endpoint` (required): The HTTP endpoint to send requests to
+- `method`: The HTTP verb to use - i.e. `GET`, `PUT`, `POST`, `DELETE`
```
Comment (morenol) [inline content/connectors/sources/http.md:19]: I think that we should mention which is the default (GET)

File: content/connectors/sources/http.md [hunk 2]
```diff
@@ -1,3 +1,99 @@
...
+- `body`: The body to use in the HTTP request to the endpoint
+- `interval`: The period (in seconds) between sending requests to the endpoint
```
Comment (morenol) [inline content/connectors/sources/http.md:21]: I think that we should mention which is the default (300 seconds)

File: content/connectors/sources/http.md [hunk 3]
```diff
@@ -1,3 +1,99 @@
...
+type: http
+topic: cat-facts
+create_topic: true
+direction: source
+parameters:
+  endpoint: https://catfact.ninja/fact
+  interval: 10
+```
+
+To run as a Managed Connector, use the `fluivo connector` command:
```
Comment (morenol) [inline content/connectors/sources/http.md:77]:
```suggestion
To run as a Managed Connector, use the `fluvio connector` command:
```

Comment (nicholastmosher) [reply content/connectors/sources/http.md:77]: Good catch!

**Review by morenol — APPROVED** [review]

**Review by simlay — APPROVED** [review]
lgtm

**Pull request merged**